Sources/ZKSync/Wallet/Wallet.swift

```diff
@@ -51,37 +96,36 @@ public protocol Wallet {
     
     func disable2FA(completion: @escaping (ZKSyncResult<Toggle2FAInfo>) -> Void) throws
     
-    func buildSignedChangePubKeyTxOnchain(fee: TransactionFee,
-                                          accountId: UInt32,
-                                          nonce: UInt32,
-                                          timeRange: TimeRange) -> Promise<SignedTransaction<ChangePubKey<ChangePubKeyOnchain>>>
-    
-    func buildSignedChangePubKeyTxSigned(fee: TransactionFee,
-                                          accountId: UInt32,
-                                          nonce: UInt32,
-                                          timeRange: TimeRange) -> Promise<SignedTransaction<ChangePubKey<ChangePubKeyECDSA>>>
-    
-    func buildSignedForcedExitTx(target: String,
-                                 tokenIdentifier: String,
-                                 fee: BigUInt,
-                                 accountId: UInt32,
-                                 nonce: UInt32,
-                                 timeRange: TimeRange) -> Promise<SignedTransaction<ForcedExit>>
-    
-    func buildSignedTransferTx(to: String,
-                               tokenIdentifier: String,
-                               amount: BigUInt,
-                               fee: BigUInt,
-                               accountId: UInt32,
-                               nonce: UInt32,
-                               timeRange: TimeRange) -> Promise<SignedTransaction<Transfer>>
-
-    func buildSignedWithdrawTx(to: String,
-                               tokenIdentifier: String,
-                               amount: BigUInt,
-                               fee: BigUInt,
-                               accountId: UInt32,
-                               nonce: UInt32,
-                               timeRange: TimeRange) -> Promise<SignedTransaction<Withdraw>>
+//    func buildSignedChangePubKeyTxOnchain(fee: TransactionFee,
+//                                          accountId: UInt32,
+//                                          nonce: UInt32,
+//                                          timeRange: TimeRange) -> Promise<SignedTransaction<ChangePubKey<ChangePubKeyOnchain>>>
+//    
+//    func buildSignedChangePubKeyTx(fee: TransactionFee,
+//                                   accountId: UInt32,
```

Comment:
Maybe remove the commented out code?

Reply:
Yep, forgot to remove this commented part. Looks like these methods should not be part of `Wallet` protocol (in Java version they're not present).